wrapanapi/systems/ec2.py

```diff
@@ -1595,3 +1596,41 @@ def cleanup_resources(self):
         self.remove_all_unused_nics()
         self.remove_all_unused_volumes()
         self.remove_all_unused_ips()
+
+    def list_ocps(self):
+        """
+        List openshift clusters (name, metadata, resources) where metadata can be used for uninstalling the cluster
+        """
+        ocp_list = []
+        resources = self.resourcegroupstaggingapi_connection.get_resources()["ResourceTagMappingList"]
+        for resource in resources:
+            tags = resource["Tags"]
```

Comment:
No need to label `tags` here if its only used in the `for` statement, just reference `resource['Tags']` directly